src/routes/__init__.py

```
@@ -5,3 +5,4 @@
 from routes.payments import router as payments_router
 from routes.orders import router as orders_router
 from routes.extra_functionality_movie import router as extra_functionality_movie_router
+from routes.comments import router as comments_router
```

Ruff (0.11.9)

`routes.comments.router` imported but unused; consider removing, adding to `__all__`, or using a redundant alias

(F401)